mcc/odin/odin/snapshot_list.go

```diff
@@ -1,12 +1,37 @@
 package odin
 
 import (
+	"fmt"
 	"sort"
+	"strings"
 
 	"github.com/aws/aws-sdk-go/service/rds"
 	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
 )
 
+const (
+	// RFC8601 is the date/time format used by AWS.
+	RFC8601 = "2006-01-02T15:04:05-07:00"
```

Comment:
But what does 15 stand for? Just %H? Don't they run out of numbers?
I am starting to like JavaScript :-)